scripts/aws.hs

```diff
@@ -1,17 +1,46 @@
-#!/usr/bin/env nix-shell
-#! nix-shell -j 4 -i runhaskell -p 'pkgs.haskellPackages.ghcWithPackages (hp: with hp; [ turtle ])'
-#! nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/f2c4af4e3bd9ecd4b1cf494270eae5fd3ca9e68c.tar.gz
```

Comment:
I can do that, no problem, but..

What if we consider entering `nix-shell` a prerequisite for working with a `iohk-nixops` checkout?

We would get this:

1. ability to drop all these nixpkgs/package set lines from all these scripts.
2. a standartized environment
3. reduced script startup times -- since they are going to share the Nix cache for their dependencies -- since the `nixpkgs` is the same across all of them _and_ whatever builds of `cardano-sl`
4. a reduced zoo of different `nixpkgs` versions used

I really don't want to come across pushy, but I don't think we ever really discussed this in a satisfying manner.